lib/registration.pm

```diff
@@ -43,6 +44,14 @@ our %SLE15_MODULES = (
     serverapp => 'Server-Applications'
 );
 
+# The expected modules of a default installation per product. Use them if they
+# are not preselected, to crosscheck or just recreate automatic selections
+# manually
+our %SLE15_DEFAULT_MODULES = (
+    sles => 'base,script,desktop,serverapp',
+    sled => 'base,script,desktop'
```

Comment:
this looks incomplete. It should contain at least productivity as well.

Reply:
true but different story. For now I am just changing the scope of the variable.